crates/chain/src/chain_data.rs

```diff
@@ -58,14 +58,16 @@ pub enum ConfirmationTime {
     /// The transaction is unconfirmed
     Unconfirmed {
         /// The last-seen timestamp in unix seconds.
-        last_seen: u64,
+        last_seen: Option<u64>,
```

Comment:
Maybe it's not worth it to put an option here either? Assuming that we agree that unconfirmed txs without a last-seen are not worth returning? The `ConfirmationTime` type is only used in `bdk_wallet` as a field in `LocalOutput` (which is a UTXO). There is no point showing a UTXO that is not part of the canonical chain right?

Reply:
> There is no point showing a UTXO that is not part of the canonical chain right?

I'm leaning towards no, a utxo owned by the wallet shouldn't qualify as a `LocalOutput` if it doesn't come from a tx in the canonical chain.